lib/jsonapi/serializer.ex

```diff
@@ -28,7 +28,7 @@ defmodule JSONAPI.Serializer do
       meta: meta
     }
 
-    merge_links(encoded_data, data, view, conn, remove_links?())
+    merge_links(encoded_data, data, view, conn, remove_links?(), true)
```

Comment:
We need to make this configurable, pagination is optional http://jsonapi.org/format/#fetching-pagination and not everyone will want it.

Reply:
That's true, but if `links` is not overridden on the view, this will have no effect, because it'll merge an empty map, right? Does it need to be more configurable than that?

Reply:
What is the suggestion here? Add an environment config like `remove_links`?

Reply:
@zamith sorry for slow response, yes similar to remove_links